src/vaults/enums/vault-status.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum VaultStatus {
+  deprecated,
```

Comment:
we will use discontinued
`deprecated <> discontinued` as annoying as it is
we can also support both but effectively we will treat it the same

Reply:
realized we have somewhat a duplicate here - might want to think about if we care to resolve them

https://github.com/Badger-Finance/badger-sdk/blob/main/src/api/enums/vault-state.enum.ts